bitnami/postgresql-ha/values-production.yaml

```diff
@@ -236,15 +236,13 @@ postgresql:
   ## ref: https://github.com/bitnami/bitnami-docker-postgresql-repmgr#configuration-file
   ##
   # repmgrConfiguration: |-
-
```

Comment:
Do not remove the empty lines

Reply:
it was done by my linter automatically, fixing it!